client/src/pages/FavoritePage.js

```diff
@@ -0,0 +1,11 @@
+import { Header } from "../components/Header";
```

Comment:
default export is the prefered method for components

so in your Header.js component it's
```
const Header = ...
...
export default Header
```

then you import like this 
```suggestion
import Header  from "../components/Header"
```

that counts for every Component :)  👩‍🎤

Reply:
Thanks! I was not so sure anymore about the differences, read up on it again now, will implement export default  for my components :)